flock: failed to execute command - No such file or directory

I have just tried to use it on macOS Mojave but can't.

```
flock -n ~/my-lock.lock -c "rsync -avEr --progress --delete-after -e \"ssh -p 5443 -o StrictHostKeyChecking=no\" ~/Documents user@192.168.1.32:/Documents/"
```

The command above says
```
flock: failed to execute command: -c: No such file or directory
```

Then tried without `-c` and got `No such file or directory` as well.

Any ideas?

Thanks

Hi,

I am trying to build the project on MAC OS Mojave and still getting this error:

flock: failed to execute command: -c: No such file or directory
make: *** [/Users/.../.gradle/caches/golang/go1.14.1.darwin-amd64.tar.gz] Error 66.

Could you please help me to resolve this.

--- a/README.md
+++ b/README.md
@@ -9,6 +9,10 @@ To exclusively lock `/tmp/my.lock` while running the utility
 `echo "hello, world!"`:
 
     flock /tmp/my.lock echo "hello, world!"
+    
+You can also use to run a single instance of rsync at a time - note: no need to escape double quotes
+
+    flock -n /tmp/my-lock.lock rsync -avr -e "ssh -p 22" "~/my path with spaces/" user@remote-host:/Documents/
 
 ## Installing
 

